app/src/pages/outside/loginPage/serviceVersionsBlock/serviceVersionsBlockWithData.jsx

```diff
@@ -3,18 +3,22 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import fetchJsonp from 'fetch-jsonp';
 import semverDiff from 'semver-diff';
+import { compositeInfoSelector } from 'controllers/appInfo';
 import { ServiceVersionsBlock } from './serviceVersionsBlock';
 
 @connect((state) => ({
   serviceVersions: state.appInfo,
```

Comment:
Remove if not needed.
Also change the `this.calculateServices` method signature